README.md

```diff
@@ -1,51 +1,118 @@
+[ZSON](docs/formats/zson.md),
+which is a superset of JSON and the relational models.
+ZSON is syntax-compatible with JSON
+but it has a comprehensive type system that you can use as little or as much as you like.
+Zed types can be used as schemas.
 
-If you don't have Go installed, download and install it from the
-[Go downloads page](https://golang.org/dl/). Go version 1.16 or later is
-required.
+Zed also has a cloud-based object design that was modeled after
+the `git` design pattern.  Commits to the lake are transactional
+and consistent.  Search index updates are also transactionally
+consistent with any ingested data and searches can run with or
+without indexes.
 
-To install the binaries in `$GOPATH/bin`, clone this repo and
-execute `make install`:
+## Quick Start
 
+_Detailed documentation [is available](docs/README.md)._
+
+The quickest way to get running on macOS, Linux, or windows
+is to download a pre-built release binary.
+You can find these binaries on the Github releases page.
+
+Once installed, you can run the query engine from the command-line using `zq`:
+```
+echo '{"s":"hello, word"}' | zq -Z -
+```
+Or you can run a Zed lake server, load it with data using `zapi`, and hit the API.
+In one shell, run the server
+```
+mkdir scratch
+cd scratch
+zed lake serve
+```
+And in another shell, run the client:
+```
+zapi create Demo
+zapi use Demo@main
+echo '{s:"hello, world"}' | zapi load -
+zapi query "from Demo"
+```
+You can also use `zed` from Python.  After you install the Zed Python:
+```
+pip install "git+https://github.com/brimdata/zed#subdirectory=python/zed"
+```
+You can hit the Zed service from a Python program:
+```python
+import zed
+
+# Connect to the REST API at the default base URL (http://127.0.0.1:9867).
+# To use a different base URL, supply it as an argument.
+client = zed.Client()
+
+# Begin executing a Zed query for all records in the pool named "Demo".
+# This returns an iterator, not a container.
+records = zed.query('from Demo'):
+
+# Stream records from the server.
+for record in records:
+    print(record)
+```
+See the [python/zed](python/zed) for more details.
+
+### Brim
+
+You can use the [Brim app](https://github.com/brimdata/brim)
+to explore, query, and shape the data in your Zed lake.
+
+We originally developed Brim for security-oriented use cases
+(having tight integration with [Zeek](https://zeek.org/),
+[Suricata](https://suricata.io/), and
+[Wireshark](https://www.wireshark.org/)),
+but we are actively extending Brim with UX for handling generic
+data sets to support data science, data engineering, and ETL use cases.
+
+### Building from Source
+
+It's also easy to build `zed` from source:
 ```
 git clone https://github.com/brimdata/zed
 cd zed
 make install
 ```
+This installs binaries in your `$GOPATH/bin`.
+
+> If you don't have Go installed, download and install it from the
+> [Go instasll page](https://golang.org/doc/install). Go version 1.16 or later is
+> required.
```

Comment:
Maybe we should also have a section for downloading and using the pre-built platform specific binaries?

Reply:
@mattnibs: There's mention of the GitHub Releases page in the "Quick Start" section, and I added a suggestion about hyperlinking it. But you make a good point. Perhaps a two-line code block could be added up there showing an example on a platform, e.g. Linux:

```
curl -OL https://github.com/brimdata/zed/releases/download/v0.30.0/zed-v0.30.0.linux-amd64.zip
unzip zed-v0.30.0.linux-amd64.zip
```

If this is desirable, I could add an item to my checklist to update this version ref each time we release.

Reply:
@otoolep put these details nicely in the Releases section.  I like how he did it.  Have a look here: https://github.com/rqlite/rqlite/releases